pkg/bundle/publish/publish.go

```diff
@@ -4,38 +4,151 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"io/ioutil"
 	"os"
 	"path/filepath"
+	"strings"
 
+	"github.com/RedHatGov/bundle/pkg/archive"
+	"github.com/RedHatGov/bundle/pkg/config"
+	"github.com/RedHatGov/bundle/pkg/config/v1alpha1"
+	"github.com/RedHatGov/bundle/pkg/image"
+	"github.com/RedHatGov/bundle/pkg/metadata/storage"
+	"github.com/google/uuid"
+	"github.com/mholt/archiver/v3"
 	"github.com/openshift/oc/pkg/cli/image/mirror"
 	"github.com/sirupsen/logrus"
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
```

Comment:
Personal nit (feel free to ignore): I like to group project module imports together at the bottom since they tend to change paths often when refactoring and does not cause reordering of a big block of imports.
```suggestion
	"github.com/google/uuid"
	"github.com/mholt/archiver/v3"
	"github.com/openshift/oc/pkg/cli/image/mirror"
	"github.com/sirupsen/logrus"
	utilerrors "k8s.io/apimachinery/pkg/util/errors"
	"k8s.io/cli-runtime/pkg/genericclioptions"

	"github.com/RedHatGov/bundle/pkg/archive"
	"github.com/RedHatGov/bundle/pkg/config"
	"github.com/RedHatGov/bundle/pkg/config/v1alpha1"
	"github.com/RedHatGov/bundle/pkg/image"
	"github.com/RedHatGov/bundle/pkg/metadata/storage"
```